config/pmd-ruleset.xml

```diff
@@ -17,6 +17,7 @@
         <exclude name="AvoidReassigningParameters" />
         <exclude name="AvoidStringBufferField" />
         <exclude name="ForLoopCanBeForeach" />
+        <exclude name="GuardLogStatement" />
```

Comment:
I will review those new pmd rules and enable later. The same with checkstyle.

Reply:
Can we create an issue for that?

Reply:
https://github.com/getodk/collect/issues/4794